### ISSUE TITLE
Stopped working today.

### ISSUE THREAD

**Issue by notque** [ZusorCode/GoodTwitter#49]
Has worked great for a long time, I think twitter may have changed their layout or something cause it stopped working. Firefox version, from the github repo. Timeline won't load.

**Comment by kylemeister1** [comment]
Having same issue, but using v2.6 installed from the official Firefox Add-Ons page.

**Comment by MohammadMSFT** [comment]
> 
> 
> Has worked great for a long time, I think twitter may have changed their layout or something cause it stopped working. Firefox version, from the github repo. Timeline won't load.

Having same issue

**Comment by Poodwaffle** [comment]
Yep, happening to me as well, both Chrome & Firefox versions are not working. Worked fine maybe 2 days ago but now it's not.

**Comment by djr47** [comment]
I disabled uBlock Origin for twitter.com, deleted twitter cookies, and reloaded, and at least right now it's working for me again.

**Comment by jayrobwilliams** [comment]
Same for me on Chrome. I'll note that I can briefly see posts when the page first starts loading and then they disappear.

**Comment by KHolley91** [comment]
I was having same issue until disabling uBlock. Now getting this:
![image](https://user-images.githubusercontent.com/34116558/81443478-1c2c8880-913b-11ea-9db5-65f44b6a231d.png) Sad :(

**Comment by TheJRC** [comment]
Well it was fun while it lasted. :(

**Comment by notque** [comment]
> I disabled uBlock Origin for twitter.com, deleted twitter cookies, and reloaded, and at least right now it's working for me again.

This worked for me too

**Comment by Mavoy** [comment]
Yeah, Twitter got stubborn on new design. I wonder what are the chances for the future of GoodTwitter. , Creating a new Twitter client using existing API looks like a way to do it. There are mobile apps, but there are also other Twitter web clients already (I can think of HootSuite but I'm sure there's more). I really don't wanna use this terrible layout :/ I'll be switching to other websites and apps as much as I can.
